index.js

```diff
@@ -1,47 +1,53 @@
 'use strict';
 const path = require('path');
-const loadJsonFile = require('load-json-file');
-const pathType = require('path-type');
-
-module.exports = (fp, opts) => {
-	if (typeof fp !== 'string') {
-		opts = fp;
-		fp = '.';
+const fs = require('fs');
+const promisify = require('util.promisify');
+const normalizePackageData = require('normalize-package-data');
```

Comment:
The `require('normalize-package-data')` hould still be inline so it's lazily required.

Reply:
require uses sync io. this is q bad thing to do for the async method

Reply:
require uses sync io. this is q bad thing to do for the async method